### DISCUSSION TITLE
Can someone please help me? my mailer server url isn't loading

### DISCUSSION THREAD

**Discussion by gudjohnsen007** [modoboa/modoboa#3133] [General]
I just installed modoboa and my dns is configured correctly ```Congratulations! You can enjoy Modoboa at https://mail.viewourinventry.com (admin:password)``` 

I have pointed my records to my dns and has been propagated since yesterday
mail IN A myipaddress
@ IN MX  mail.viewourinventry.com

Can anyone help me? Thanks in advance

**Comment by JohnPBee** [comment]
Hi, I just went through the installation and encountered a similar issue - no response in browser.
As it turns out, the installer would not process a proper SSL certificate, and nginx would not start as a result.

That said, I managed to half-haphazardly persuade things back into action by commenting out the ssl portion from the nginx configuration file, and then manually installing a letsencrypt cert using certbot - test nginx, uncomment, reboot, everything worked

- hope this helps

**Comment by gudjohnsen007** [comment]
Thanks
